app/models/interview.rb

```diff
@@ -54,7 +54,11 @@ def information(options = {})
   end
 
   def pending?
-    !completed
+    !completed && !saved_for_later
+  end
+
+  def saved_for_later?
+    extract_associated(:application_submission).saved_for_later?
```

Comment:
We only have one associated `:application_submission` (right?), so using `extract_associated` provides no performance benefits (the preload might actually make this slower- my `ActiveRecord` knowledge is a bit fuzzy right now).

the simpler `delegate :saved_for_later?, to: :application_submission` in the header would suffice.